src/test/java/reposense/parser/ArgsParserTest.java

```diff
@@ -57,23 +57,25 @@ public void parse_withExtraWhitespaces_success() throws ParseException, IOExcept
     }
 
     @Test
-    public void parse_configFileOnly_success() throws ParseException, IOException {
+    public void parse_configFileOnly_success() throws ParseException, IOException, java.text.ParseException {
         Path expected = CONFIG_FILE_ABSOLUTE;
         String input = String.format("-config %s", CONFIG_FILE_ABSOLUTE);
         CliArguments cliArguments = ArgsParser.parse(translateCommandline(input));
         Files.isSameFile(expected, cliArguments.getConfigFilePath());
-        //Optional arguments have default values
+        // Optional arguments have default values
         Assert.assertEquals(Optional.empty(), cliArguments.getSinceDate());
         Assert.assertEquals(Optional.empty(), cliArguments.getUntilDate());
-        Assert.assertTrue(Files.isSameFile(Paths.get("."), cliArguments.getOutputFilePath()));
+        // Checks if output folder generated is in correct format
+        ArgsParser.DEFAULT_REPORT_NAME_FORMAT.parse(cliArguments.getOutputFilePath().getFileName().toString());
 
         input = String.format("-config %s", CONFIG_FILE_RELATIVE);
         cliArguments = ArgsParser.parse(translateCommandline(input));
         Files.isSameFile(expected, cliArguments.getConfigFilePath());
-        //Optional arguments have default values
+        // Optional arguments have default values
         Assert.assertEquals(Optional.empty(), cliArguments.getSinceDate());
         Assert.assertEquals(Optional.empty(), cliArguments.getUntilDate());
-        Assert.assertTrue(Files.isSameFile(Paths.get("."), cliArguments.getOutputFilePath()));
+        // Checks if output folder generated is in correct format
+        ArgsParser.DEFAULT_REPORT_NAME_FORMAT.parse(cliArguments.getOutputFilePath().getFileName().toString());
```

Comment:
https://docs.oracle.com/javase/9/docs/api/java/text/SimpleDateFormat.html#parse-java.lang.String-java.text.ParsePosition-

I think you need to additionally check that the result is not null.